kythe/go/indexer/testdata/meta.go

```diff
@@ -13,4 +14,5 @@ func Foobar() {}
 //- FA.loc/end   25
 //- FA defines/binding Foobar
 //- Foobar.node/kind function
-//- _Alt=vname(gsig, gcorp, groot, gpath, glang) generates Foobar
+//- _Alt=vname(gsig, gcorp, groot, gpath, glang) generates Foobar?
+//- vname("", gcorp, groot, gpath, "") generates vname("", kythe, _, "go/indexer/metadata_test", "")
```

Comment:
This test should be expanded to include multiple definitions to catch edge duplication.